yang/ietf-l3vpn-ntw.yang

```diff
@@ -1933,6 +1933,41 @@ module ietf-l3vpn-ntw {
                       }
                       uses vpn-common:service-status;
                     }
+                    container igmp {
+                      when "derived-from-or-self(../type, "
+                         + "'vpn-common:igmp')" {
+                        description
+                          "Only applies when protocol is IGMP.";
+                      }
+                      if-feature "vpn-common:rtg-igmp";
+                    leaf subscription_channel {
+                      type inet:ip-address;
+                      description
+                        "IGMP subscription channel.";
+                    }
+                    leaf version {
```

Comment:
These need to be defined as identities so that we can easily support future versions (if any)